tern/analyze/docker/dockerfile.py

```diff
@@ -333,3 +334,47 @@ def write_locked_dockerfile(dfile, destination=None):
         file_name = constants.locked_dockerfile
     with open(file_name, 'w') as f:
         f.write(dfile)
+
+
+def check_multistage_dockerfile(dfobj):
+    """Given a dockerfile object, return if it is a multistage dockerfile,
```

Comment:
Do you see yourself using the True/False return value here? It seems somewhat redundant. The only other place `check_multistage_dockerfile` is called in the PR, you throw away the True/False value. Users of this function will know it is a multistage dockerfile if the length of the list returned is > 1.

Reply:
No. I think this is redundant here. I will remove this.